stats/sink.go

```diff
@@ -56,9 +56,11 @@ func (c *CounterSink) Add(s Sample) {
 func (c *CounterSink) Calc() {}
 
 func (c *CounterSink) Format(t time.Duration) map[string]float64 {
+	rps := c.Value / (float64(t) / float64(time.Second))
 	return map[string]float64{
 		"count": c.Value,
-		"rate":  c.Value / (float64(t) / float64(time.Second)),
+		"rate":  rps,
+		"rps":   rps,
```

Comment:
I finally read through all discussion on Slack, and naming this "rps" sounds confusing to me. Like you guys were discussing, RPS implies "request per second", while "rate per second" is meaningless... If anything this should probably be "cps"/"count per second".

I don't want to start a huge discussion about such a nitpick again, but as a user I would find "rps" even more confusing than "rate". At least with "rate" we can document that this means "counts per second", instead of trying to explain what "Counter.rps" means.

Reply:
I tend to mostly agree with you, but not too strongly, so I'm leaving this open and you, @MStoykov and @esquonk can reach a consensus next week and decide if you should merge this or not.

Reply:
hmm actually, thinking about this some more, I am withdrawing this PR. Completely ignoring whether `Counter.rps` is easier to grok than `Counter.rate`, adding `Counter.rps` in k6 v0.28.0 when it wasn't present in old k6 versions is a problem and for sure an extra complexity... So, I am again back on the opinion we should simply document `rate` better and fix the whole issue with better names in thresholds v2.

Reply:
If this is an issue for k6 then I guess we'll have to stick with `rate`, although I would prefer to have a different, less-ambiguous name name (`cps`?). Or drop this method completely.

Reply:
We can't _not_ have rate, given that we've had it and documented it for so long, and having another one, even with the perfect name, will just add extra confusion. We should document its peculiarity, support it everywhere, and figure out a better alternative in the future, when we're not burdened by the limitations we currently have.

Reply:
Yeah, I don't see a "perfect" solution here... let's leave this till Monday, maybe a fresh idea will present itself :smile: